src/vorta/models.py

```diff
@@ -106,6 +106,9 @@ class Meta:
 class SourceFileModel(pw.Model):
     """A folder to be backed up, related to a Backup Configuration."""
     dir = pw.CharField()
```

Comment:
Change this name to dirPath?

Reply:
Don't, it will require a very awkward db upgrade.